bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownButton.html

```diff
@@ -4,15 +4,16 @@
     </head>
     <body>
         <wicket:panel>
-            <a class="btn dropdown-toggle" data-toggle="dropdown" wicket:id="btn" href="#">
-                <span wicket:id="icon"></span>
-                <wicket:container wicket:id="label"></wicket:container>
-            </a>
-            <ul wicket:id="dropdown-menu" class="dropdown-menu">
-                <li wicket:id="buttons">
-                    <a wicket:id="button" class="dropdown-item"></a>
-                </li>
-            </ul>
+           <a class="dropdown-toggle" wicket:id="btn" href="#" type="button" 
+		  		data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+		  		<span wicket:id="icon"></span>
+		    	<wicket:container wicket:id="label"></wicket:container>
+		  </a>
+		  <div wicket:id="dropdown-menu" class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+		  	<div wicket:id="buttons">
+		    	<a wicket:id="button" class="dropdown-item" href="#">Action</a>
```

Comment:
Same for `href="#"`.

Reply:
Is the text `Action` only for a preview here ?

Reply:
- href="#" exchanged against href="javascript:;" 
- Action removed (just needed for my project)